src/snowflake/snowpark/_internal/server_connection.py

```diff
@@ -666,6 +669,7 @@ def get_result_set(
                     num_statements=len(main_queries),
                     params=params,
                     ignore_results=ignore_results,
+                    async_post_actions=post_actions,
```

Comment:
is that fixing a bug?

Reply:
yes indeed.

Reply:
https://github.com/snowflakedb/snowpark-python/pull/2405/files#diff-a7f9cc0095b005bbd89b9100dc2d75b6b26c7ad70923bc502917ee46c1989bb4R449-R452

Reply:
earlier it would pull post_actions from snowflake plan which may not have drop table commands coming from large query breakdown